File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.java
class Solution {
    public int solution(int[] arr1, int[] arr2) {
        int sum1=0;
        int sum2=0;
        
        for(int i: arr1){
            sum1+=i;
        }
        
        for(int i: arr2){
            sum2+=i;
        }
                
        if(arr1.length < arr2.length){
            return -1;
        }
        else if(arr1.length > arr2.length){
            return 1;
        }
        else{
            if(sum1 < sum2){
                return -1;
            }
            else if(sum1 > sum2){
                return 1;
            }
            else{
                return 0;
            }
        }
        
    }
}

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.py
def solution(arr1, arr2):
    answer = 0
    
    if len(arr1)!=len(arr2) :
        answer=1 if len(arr1) > len(arr2) else -1
    else : 
        if sum(arr1) == sum(arr2) :
            answer = 0
        elif sum(arr1) > sum(arr2) :
            answer = 1
        else : answer = -1
        
   
    return answer